script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -631,7 +644,7 @@ That said, it's not as though each and every one of them has it out for humans,
 [sha:a032975dc854c46f8c1c111510704f358ff3ddfc]{
 -- Page 616, Offset 18126.
 -- 「『二十七祖』というのは在り方を示す呼称だから変わらないけど、実際にはえーと……残り二十一祖かな？
-"The phrase 'The Twenty-Seven Ancestors' is more a reference to the way things %{i}ought%{/i} to be than the way things %{i}are%{/i},
+"'The Twenty-Seven Ancestors' indicates the way they are, rather than their number.
```

Comment:
Can you elaborate on what 'the way they are' _means_ here? I assume there are 27 'principles' or whatever, and there's supposed to be one ancestor for each? Depending on the answer I might tweak this a bit, just because 'the way they are' is kinda vague (though, yeah, so might be 在り方)